Libplanet.Tools/Mpt.cs

```diff
@@ -23,6 +23,14 @@ internal enum SchemeType
 
     public class Mpt
     {
+        private const string KeyValueStoreURIExample =
+            "(e.g., <kv-store-type>://<kv-store-path>, rocksdb:///path/to/kv-store)";
+
+        private const string KeyValueStoreArgumentDescription =
```

Comment:
Repeating the term “key value store” feels quite redundant.  I suggest we use the term “kv store” instead for this meaning consistently.

```suggestion
        private const string KVStoreArgumentDescription =
```